draft-ietf-dprive-dnsoquic.md

```diff
@@ -783,8 +783,21 @@ this include that DoQ
 
 # Security Considerations
 
+A general Threat Analysis of the Domain Name System is found in {{?RFC3833}}.
+This analysis was written before the development of DoT, DoH and DoQ, and
+probably needs to be updated.
+
 The security considerations of DoQ should be comparable to those of DoT
-{{?RFC7858}}.
+{{?RFC7858}}. DoT as specified in {{?RFC7858}} only addresses the stub to
+recursive resolver scenario, but the considerations about person-in-the-middle
+attacks, middleboxes and caching of data from clear text connections also
+apply for DoQ to the resolver to authoritative server scenario. 
+As stated in {{authentication}} the authentication requirements for securing zone transfer using DoQ are the same as those for zone transfer over DoT, therefore the general security considerations are entirely analogous to those described in {{!RFC9103}}.
+
+DoQ relies on QUIC, which itself relies on TLS 1.3 and thus supports by default
+the protections against downgrade attacks described in {{?BCP195}}.
```

Comment:
Additional downgrade protections are documented in the version negotiation draft.

Reply:
Good point.